projects/coffea-experiment-support.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Coffea development for LHC experiments
+
+postdate: 2022-02-22
+categories:
+  - Analysis tools
+durations:
+  - 3 months
+experiments:
+  - CMS
+  - ATLAS
+skillset:
+  - C++
+  - Python
+  - CI/CD
+  - Linux
+  - Git
+status:
+  - Available
+project:
+  - IRIS-HEP
+location:
+  - Any
+commitment:
+  - Full time
+program:
+  - IRIS-HEP fellow
+
+shortdescription: Development of coffea to support multiple LHC experiment analysis workflows
+
+description: >
```

Comment:
@lgray @nsmith- I had forgotten to take care of this last week and just remembered, so I just threw something together here before bed so we can get this up, but if you can give this a look over and add comments or revisions that would be great.

Reply:
Looks good to me